pkg/odo/cli/dev/dev.go

```diff
@@ -292,7 +294,8 @@ It forwards endpoints with exposure values 'public' or 'internal' to a port on l
 			genericclioptions.GenericRun(o, cmd, args)
 		},
 	}
-	devCmd.Flags().BoolVarP(&o.randomPorts, "random-ports", "f", false, "Assign random ports to redirected ports")
+	devCmd.Flags().BoolVar(&o.randomPortsFlag, "random-ports", false, "Assign random ports to redirected ports")
+	devCmd.Flags().BoolVar(&o.debugFlag, "debug", false, "Execute the debug command to start the component inside container")
```

Comment:
Bit of an odd sentence. Maybe: `Excecute the debug command within the component` ?